client/landing/stepper/declarative-flow/internals/steps-repository/unified-plans/index.tsx

```diff
@@ -84,6 +87,12 @@ type ProvidedDependencies = {
 const PlansStepAdaptor: StepType< {
 	submits: ProvidedDependencies;
```

Comment:
You can add `accepts` here, adjacent to submits to add accepted props.
```suggestion
	submits: ProvidedDependencies;
	accepts: {
		isInSignup?: boolean;
		isStepperUpgradeFlow?: boolean;
		selectedFeature?: string;
	};
```